cts/cts-cli.in

```diff
@@ -1718,6 +1743,12 @@ for t in $tests; do
     esac
 done
 
+if [ -x /usr/bin/xmllint ]; then
+    validate=1
+else
+    echo "/usr/bin/xmllint is missing - install it to validate command output"
+fi
```

Comment:
We usually check $(which xmllint) to allow for local environment flexibility. Using a variable for the xmllint location would help avoid the previous problem too